ur_robot_driver/launch/ur_control.launch.py

```diff
@@ -0,0 +1,560 @@
+# Copyright (c) 2021 PickNik, Inc.
```

Comment:
Since we don't have launchfiles in `ur_robot_driver` on galactic, I guess it doesn't make much sense to add this here.